Makefile

```diff
@@ -386,21 +418,41 @@ OPERATOR_SDK = $(shell which operator-sdk)
 endif
 endif
 
-# Generate bundle manifests and metadata, then validate generated files.
-.PHONY: bundle
-bundle: manifests kustomize operator-sdk
-	$(OPERATOR_SDK) generate kustomize manifests -q
+##@ Deployment
+
+.PHONY: install
+install: manifests kustomize ## Install CRDs into a cluster.
+	$(KUSTOMIZE) build config/crd | $(CLUSTER_CLIENT) apply -f -
+
+.PHONY: uninstall
+uninstall: manifests kustomize ## Uninstall CRDs from a cluster.
+	- $(KUSTOMIZE) build config/crd | $(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f -
+
+.PHONY: predeploy
+predeploy: ## Prepare for deployment.
```

Comment:
I think we can remove the comments here since I believe its not meant to be called by itself?

Reply:
Sounds good.